### ISSUE TITLE
Move usecases to external resources

### ISSUE THREAD

**Issue by pauleve** [colomoto/colomoto-docker#65]
The docker repository should not aim at including usecases. Usecases should be hosted on external resources (like gist or zenodo) and referenced on the website at https://colomoto.github.io/colomoto-docker/#tutorials-and-use-cases.

- [x] migrate and reference 'Usecase - Mutations enabling tumour invasion.ipynb' (https://nbviewer.jupyter.org/gist/pauleve/a86717b0ae8750440dd589f778db428f/Usecase%20-%20Mutations%20enabling%20tumour%20invasion.ipynb)
- [ ] migrate and reference 'Usecase - Balance of Th17 vs Treg cell populations.ipynb'
- [ ] migrate and reference 'Population-level Cell Fate with UpPMaBoSS'
- [ ] migrate and reference snakemake?
- [ ] delete 'usecase' directory

**Comment by aurelien-naldi** [comment]
Agreed, but I think we need to prepare a better system to host them first. Using gist and common notebook viewers is fine, but I would rather keep them together in a place we can handle.

I would add usecases to a separate git repository along with the required files and some metadata (to be defined!). Then we can build a set of static pages to host the use cases with a web preview (using hugo and nbconvert) and download option. We can even annotate them with versions of the colomoto image for which they have been tested and use that to open them in mybinder.

We can start playing with this in a new repository with pages hosted on github (I can set this up) and remove them from the docker repository when we are happy with the result.

**Comment by pauleve** [comment]
So you see them as some sort of official use cases? In general we can think that they are supplementary material for publications, so it is mainly up to the authors to manage how they are hosted and possibly updated with time. 

But we can do both and have some sort of advanced tutorials hosted on a colomoto-notebooks or colomoto-usecases repository, indeed.

**Comment by aurelien-naldi** [comment]
Not really official, rather a repository open to submission by modellers. Of course it can also contain some "official" notebooks (in particular tutorials). For me it would be nice to have a go-to place to submit analysis notebooks and it would be convenient for new users to have an obvious place to search for example code.

I think that this kind of repository, along with a model repository (on which I am planning to come back once we learn a bit more from the GINsim one) would be a big selling point for the colomoto community.

**Comment by pauleve** [comment]
:+1:
